src/Private/Helpers.ps1

```diff
@@ -1362,6 +1349,20 @@ function ConvertTo-PodeResponseContent {
             }
         }
 
+        { $_ -ilike '*/yaml' } {
```

Comment:
This'll also need to support `*/x-yaml`, as it is/was the main content-type for YAML.